src/test/java/com/thealgorithms/geometry/BresenhamLineTest.java

```diff
@@ -0,0 +1,87 @@
+package com.thealgorithms.geometry;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.awt.Point;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+/**
+ * The {@code BresenhamLineTest} class contains unit tests for the {@code BresenhamLine} class.
+ * It verifies the correctness of the findLine method, which generates a list of points
+ * representing a straight line between two specified endpoints.
+ *
+ * <p>This test class uses JUnit 5 for testing and covers various scenarios including horizontal,
+ * vertical, and diagonal lines, as well as lines with negative coordinates.</p>
+ */
+class BresenhamLineTest {
```

Comment:
Will be better to use Parametrized Tests for this class. Because all tests are the similar, only test data different